DiagnosticsExtension/Controllers/ConnectionStringValidationController.cs

```diff
@@ -85,5 +115,23 @@ public async Task<HttpResponseMessage> ValidateAppSetting(string appSettingName,
                 return Request.CreateErrorResponse(HttpStatusCode.NotFound, $"AppSetting {appSettingName} not found");
             }
         }
+
+        [HttpGet]
+        [Route("test")]
+        public async Task<IHttpActionResult> Test()
```

Comment:
>public async Task<IHttpActionResult> Test() [](http://example.com/codeflow?start=8&length=43)

Is this only for testing or do you want to leave this controller in production ?

Reply:
Accidentally checked testing code in, will remove it

Reply:
removed